gneiss/regression/_transformer.py

```diff
@@ -0,0 +1,33 @@
+from ._ols import OLSModel
+from ._mixedlm import LMEModel
+from ._format import RegressionFormat_g
+from gneiss.plugin_setup import plugin
+
+
+@plugin.register_transformer
+def _1(data: OLSModel) -> RegressionFormat_g:
+    ff = RegressionFormat_g()
+    with ff.open() as fh:
+        data.write_pickle(fh)
+    return ff
+
+
+@plugin.register_transformer
+def _2(ff: RegressionFormat_g) -> OLSModel:
+    with ff.open() as fh:
+        print(fh)
+        return OLSModel.read_pickle(fh)
+
+
+@plugin.register_transformer
+def _3(data: LMEModel) -> RegressionFormat_g:
+    ff = RegressionFormat_g()
+    with ff.open() as fh:
+        data.write_pickle(fh)
+    return ff
+
+
+@plugin.register_transformer
```

Comment:
Would it make sense to have a transform from `LMEModel` -> `OLSModel`? It is possible now indirectly via `RegressionFormat_g` but that doesn't mean it's something that should be done...